File: iniciante/URI1040.java
package URI;

import java.util.Scanner;

public class URI1040 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);


        double n1 = scanner.nextDouble() * 2;
        double n2 = scanner.nextDouble() * 3;
        double n3 = scanner.nextDouble() * 4;
        double n4 = scanner.nextDouble();

        double media = (n1 + n2 + n3 + n4) / 10;

        System.out.printf("Media: %.1f\n", media);

        if (media >= 5.0 && media <= 6.9) {
            System.out.println("Aluno em exame.");
            double exam = scanner.nextDouble();
            System.out.printf("Nota do exame: %.1f\n", exam);
            double finalmedia = (media + exam) / 2;

            if (finalmedia > 5.0) {
                System.out.println("Aluno aprovado.");
            } else {
                System.out.println("Aluno reprovado.");
            }
            System.out.printf("Media final: %.1f", finalmedia);
        } else if (media > 7.0) {
            System.out.println("Aluno aprovado.");
        } else if (media < 5.0) {
            System.out.println("Aluno reprovado.");
        }
    }
}

File: python/Iniciante/URI1040.py
linha = input().split()
n1, n2, n3, n4 = list(map(float, linha))

n1 = n1 * 2
n2 = n2 * 3
n3 = n3 * 4
n4 = n4 * 1

media = (n1 + n2 + n3 + n4) / 10

print(f"Media: {media:.1f}")
if media >= 7.0:
  print("Aluno aprovado.")
elif media < 7.0 and media >= 5.0:
  print("Aluno em exame.")
  exame = float(input())
  print(f"Nota do exame: {exame}")
  media = (media + exame) / 2
  if media >= 5.0:
    print("Aluno aprovado.")
  else:
    print("Aluno reprovado.")
  print(f"Media final: {media:.1f}")
else:
  print("Aluno reprovado.")